libbeat/docs/outputconfig.asciidoc

```diff
@@ -79,7 +79,7 @@ output.elasticsearch:
 
 If the Elasticsearch nodes are defined by `IP:PORT`, then add `protocol: https` to the yaml file.
 
-[source,yaml]
+["source","yaml",subs="attributes,callouts"]
```

Comment:
FYI: This line is unrelated to this PR, looks like a bug:
<img width="775" alt="Screen Shot 2019-05-02 at 12 05 08 PM" src="https://user-images.githubusercontent.com/5618806/57099963-8f9eed00-6cd2-11e9-8389-a98c910ffb5e.png">